src/HomePage/Skills/Skills.scss

```diff
@@ -1,17 +1,29 @@
-.skills {
-  display: flex;
-  justify-content: space-around;
-  flex-flow: wrap;
-  max-width: 550px;
-}
+.skills-section {
+  background: rgb(219, 217, 217);
```

Comment:
You've got a bit of a gutter around all your components which makes anything without a white background look a bit weird. I suggest setting body margin to 0 so this looks nicer (probably in your app.css file or whatever is your highest level css file)

Reply:
Have added in too